src/components/input-group/daikin-input-group.ts

```diff
@@ -0,0 +1,156 @@
+import { colorFeedbackNegative } from '@daikin-oss/dds-tokens/js/daikin/Light/variables.js';
+import { LitElement, html, unsafeCSS, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import tailwindStyles from '../../tailwind.css';
+import ctl from '@netlify/classnames-template-literals';
+
+const inputGroupContainer = ctl(`
+  flex
+  flex-col
+  justify-center
+  w-max
+  gap-2
+  font-daikinSerif
+  `);
+
+const inputGroupError = ctl(`
+    flex
+    gap-2
+    leading-[22px]
+
+    input-group-bottom-text-error
```

Comment:
```suggestion
    color-[--color-feedback-negative]
```

with

```
:host {
  --color-feedback-negative: ${unsafeCSS(colorFeedbackNegative)};
}
```

might be better